### PR TITLE
Removed quizzes sorting and searching

### PR DESCRIPTION
**Pull request by sergiy17**
[Link to JIRA ticket](http://ssu-jira.softserveinc.com/browse/LR-214)

### Author.
Sergiy Parkhuts

- [x] I have added link to this PR to the JIRA
- [x] I have logged time in JIRA.

### Reviewers.

- [x] @superNazo 
- [x] @OKrain 
- [x] @vovka

### Issue summary.

*Removed quizzes sorting and searching*

### Testing approach.

Manual testing

### CODE REVIEW

File: app/assets/javascripts/angularjs_app/components/quizzes/views/indexQuiz.html [hunk 1]
```diff
@@ -1,24 +1,6 @@
...
-  <div class="btn btn-primary" ng-click="sort('name')">Name
-    <span class="glyphicon sort-icon" ng-show="sortKey=='name'" ng-class="{'glyphicon-chevron-up':reverse,'glyphicon-chevron-down':!reverse}"></span>
-  </div>
-  <div class="btn btn-primary" ng-click="sort('created_at')">Date
-    <span class="glyphicon sort-icon" ng-show="sortKey=='created_at'" ng-class="{'glyphicon-chevron-up':reverse,'glyphicon-chevron-down':!reverse}"></span>
-  </div>
-</div>
 <ul class="list-group">
-  <li ng-repeat="quiz in quizzes | orderBy:sortKey:reverse | filter:search" class="list-group-item">
+  <li ng-repeat="quiz in quizzes | orderBy:sortKey:reverse" class="list-group-item">
```
Comment (superNazo) [inline app/assets/javascripts/angularjs_app/components/quizzes/views/indexQuiz.html]: if u remove all filters u also shoud remove orderBy prop

**Comment by OKrain** [comment]
:+1:

**Comment by vovka** [comment]
Looks good to me. But we can't merge while master is broken.

**Comment by vovka** [comment]
@sergiy17 please, rebase this branch

File: app/assets/javascripts/angularjs_app/components/quizzes/quizzesControllers.js [hunk 2]
```diff
@@ -32,11 +32,6 @@ quizzesControllers.controller('indexQuizCtrl',
 
     $scope.quizzes = Quizzes.query();
 
-    $scope.sort = function(keyname) {
-      $scope.sortKey = keyname;
```
Comment (vovka) [inline app/assets/javascripts/angularjs_app/components/quizzes/quizzesControllers.js]: Codacy Issue found: [Identifier 'questions_attributes' is not in camel case.](https://www.codacy.com/app/vovka/video_quiz/file/2515858177/issues/source?bid=3205144&fileBranchId=3269861#l18)

File: app/assets/javascripts/angularjs_app/components/quizzes/quizzesControllers.js [hunk 3]
```diff
@@ -32,11 +32,6 @@ quizzesControllers.controller('indexQuizCtrl',
...
-      $scope.reverse = !$scope.reverse;
```
Comment (vovka) [inline app/assets/javascripts/angularjs_app/components/quizzes/quizzesControllers.js]: Codacy Issue found: [Identifier 'questions_attributes' is not in camel case.](https://www.codacy.com/app/vovka/video_quiz/file/2515858177/issues/source?bid=3205144&fileBranchId=3269861#l19)

**Comment by vovka** [comment]
Can be merged.

**Pull request merged**